sinks/librato.py

```diff
@@ -181,6 +186,10 @@ def add_measure(self, key, value, time):
         if self.prefix:
             name = "%s.%s" % (self.prefix, name)
 
+        # Add a source prefix
+        if self.source_prefix:
+            source = "%s%s" % (self.source_prefix, source)
```

Comment:
Might be a bit confusing that the metric `prefix` uses a period delimiter while the `source_prefix` does not?